mungegithub/mungers/publisher.go

```diff
@@ -57,6 +57,8 @@ type repoRules struct {
 	srcToDst []branchRule
 	// if empty (e.g., for client-go), publisher will use its default publish script
 	publishScript string
+	// not updated when true
+	skipped bool
```

Comment:
When is `skipped` used?

Reply:
To keep the existing repos during debugging. If something breaks in sample-apiserver, you don't want to wait 30 min for the repos before that.